python/lib/cloudutils/serviceConfigServer.py

```diff
@@ -107,7 +107,7 @@ def checkHostName():
         bash("chown cloud.cloud /var/run/cloudstack-management.pid")
         #distro like sl 6.1 needs this folder, or tomcat6 failed to start
```

Comment:
@DaanHoogland No, this should be an OS capable of running the management server.